scheduler/src/cook/config_incremental.clj

```diff
@@ -99,10 +99,18 @@
   (select-config-from-values uuid (read-config config-key)))
 
 (defn resolve-incremental-config
-  "Resolve an incremental config to the appropriate value."
-  [^UUID uuid incremental-config]
-  (cond
-    (coll? incremental-config)
-    (select-config-from-values uuid incremental-config)
-    (keyword? incremental-config)
-    (select-config-from-key uuid incremental-config)))
\ No newline at end of file
+  "Resolve an incremental config to the appropriate value.
```

Comment:
Could you comment on the shape of an incremental config? What does the data shape look like?